moderncvbodyv.sty

```diff
@@ -67,12 +72,21 @@
   % regular lengths
   \setlength{\parskip}{0\p@}}
 
+\newcommand{\calculatecompensation}[1]{%
+  % calculate the number of lines used by sections and subsections
+  \settowidth{\thissectionwidth}{#1}%
+  \renewcommand{\numoflines}{\numexpr\thissectionwidth/\hintscolumnwidth\relax}%
+  \setlength{\multilinecompensationlength}{\dimexpr\hintscolumnwidth*\numoflines\relax}%
```

Comment:
I'm confused by this :confused:; why are we using `\hintscolumnwidth` times number of lines here?

Reply:
This is the strange part... From a testing perspective this helped to solve the MWE from above. 
I should have a closer look again on that issue, because I am not completely sure why this happens at all. 
I agree that we should not fix a strange behaviour with a strange workaround.